modules/core/src/test/java/org/apache/ignite/internal/client/thin/ThinClientAbstractPartitionAwarenessTest.java

```diff
@@ -389,4 +399,14 @@ public boolean isClosed() {
             return cfg.getAddresses().toString();
         }
     }
+
+    /**
+     * Excludes node if its consistent id ends with 'Test1'.
+     */
+    protected static final class ConsistentIdNodeFilter implements IgnitePredicate<ClusterNode> {
```

Comment:
There are three almost identical classes. Can we use just and and pass consistent id suffix as constructor parameter?

Reply:
We check filters equality for a cache group only by their classes. A parametrized filter is a "bad example" for a user.